hrstaffdir/tests/phpunit/WebTest/HRStaffDir/HRStaffDirProfile.php

```diff
@@ -24,7 +24,6 @@
  +--------------------------------------------------------------------+
 */
 
-require_once 'CiviTest/CiviSeleniumTestCase.php';
```

Comment:
yes I didn't run any web test .. just the unit tests ... I said its just removing the require_one so no need to waste some time on that . but thanks ... I will update the code